docs/contributing/03-stories/story-16.md

```diff
@@ -0,0 +1,67 @@
+# User Story 16 - Wing for TypeScript
+
+> **Status**: Draft
+
+Anders loves TypeScript, but he also loves the amazing experience Wing offers for building cloud applications.
+
+It's Andres' lucky day because Wing Cloud had just announced that very soon TypeScript developers are able to start leveraging Wing libraries from the comfort of
+their favorite programming language.
+
+He got an invite for a pre-release. The invite explains that initially, there is only support for `cloud.Bucket` and `cloud.Function` and for consuming [winglibs](https://github.com/winglang/winglibs).
+
+The example looks like this:
+
+First, install some libraries:
+
+```sh
+$ npm i winglang wingsdk @winglibs/cloud @winglibs/checks
+```
+
+```ts
+// main.ts
+import { inflight, App } from "wingsdk";
+import * as cloud from "@winglibs/cloud";
```

Comment:
What does the @winglibs/cloud library look like?

Reply:
It is a simple shim that exports a Bucket and Function classes with an API that mimics the Wing Cloud Library and proxies everything to the "real" resources.

This is the proposed mechanism to initially expose Wing Cloud Library resources to TypeScript users.

Eventually we will rewrite the entire SDK in winglang and then this additional layer won't be needed.

For the very first release, we can just expose these two types as an exercise. Then we can easily add the rest of the cloud library.

Reply:
@MarkMcCulloh @staycoolcall911 added a section with some example.